src/Widgets/DisplayWidget.vala

```diff
@@ -205,7 +205,9 @@ public class Display.DisplayWidget : Gtk.EventBox {
         destroy.connect (() => display_window.destroy ());
 
         use_switch.notify["active"].connect (() => {
-            //output_info.set_active (use_switch.active);
+            if (!virtual_monitor.set_active (use_switch.active)) {
+                return;
+            };
             resolution_combobox.sensitive = use_switch.active;
             rotation_combobox.sensitive = use_switch.active;
             refresh_combobox.sensitive = use_switch.active;
```

Comment:
It would be better to bind these properties to `virtual_monitor.is_active`.

Reply:
hmm, I tried replacing these in the same fashion as the use_switch.sensitive property but I couldn't get it to work, e.g:
` virtual_monitor.bind_property ("is_active", rotation_combobox, "sensitive");
`
When I tested this, the boxes wouldn't toggle visibility with the active button; they stayed stuck on their initial state. 

To be honest I've already spent wayy more time on this PR than I was expecting, so I'm tempted to punt this issue aside. What do you think?
In the mean time I've changed those lines to at least be set from virtual_monitor.is_active.

Reply:
You have to replace underscores with hyphen so:
```vala
virtual_monitor.bind_property ("is-active", rotation_combobox, "sensitive");
```